CMakeLists.txt

```diff
@@ -113,87 +126,91 @@ set(CMAKE_C_COMPILER $ENV{CMAKE_C_COMPILER})
 set(BUILDDIR "$ENV{BUILDDIR}")
 
 if(ARCH_TYPE_OPTION)
-  if(NOT
-     (${ARCH_TYPE_OPTION} STREQUAL "x86")
-     AND NOT (${ARCH_TYPE_OPTION} STREQUAL "armv7"))
-    message(
-      AUTHOR_WARNING
-        "HW architecture is not defined, using x86. Allowed values are x86/armv7 (case sensitive)"
-      )
-    set(ARCH_TYPE_OPTION "x86")
-  endif()
+    if(NOT
+       (${ARCH_TYPE_OPTION} STREQUAL "x86")
+       AND NOT (${ARCH_TYPE_OPTION} STREQUAL "armv7"))
+        message(
+            AUTHOR_WARNING
+                "HW architecture is not defined, using x86. Allowed values are x86/armv7 (case sensitive)"
+            
+)
+        set(ARCH_TYPE_OPTION "x86")
+    endif()
 else()
-  set(ARCH_TYPE_OPTION "x86")
+    set(ARCH_TYPE_OPTION "x86")
 endif()
 
 if(HMI_TYPE_OPTION)
-  if(${HMI_TYPE_OPTION} STREQUAL "NONE")
-    message(STATUS "Jenkins integration: select HMI none")
-    set(HMI "no")
-  else()
-    message(STATUS "Jenkins integration: select HTML5 HMI")
-    set(HMI "web")
-  endif()
+    if(${HMI_TYPE_OPTION} STREQUAL "NONE")
+        message(STATUS "Jenkins integration: select HMI none")
+        set(HMI "no")
+    else()
+        message(STATUS "Jenkins integration: select HTML5 HMI")
+        set(HMI "web")
+    endif()
 endif()
 
 if(MEDIA_MODE_OPTION)
-  if(${MEDIA_MODE_OPTION} STREQUAL "EXTENDED_MEDIA")
-    message(STATUS "Jenkins integration: select extended media mode")
-    set(EXTENDED_MEDIA_MODE ON)
-  endif()
+    if(${MEDIA_MODE_OPTION} STREQUAL "EXTENDED_MEDIA")
+        message(STATUS "Jenkins integration: select extended media mode")
+        set(EXTENDED_MEDIA_MODE ON)
+    endif()
 endif()
 
 if(NOT CMAKE_BUILD_TYPE)
-  set(
-    CMAKE_BUILD_TYPE Debug
-    CACHE
-      STRING
-      "Choose the type of build. Options are: None, Debug, Release, RelWithDebInfo, MinSizeRel."
-    FORCE)
+    set(
```

Comment:
@vilyukh bring to the form as 
```
set(CMAKE_BUILD_TYPE Debug
      CACHE 
              STRING 
              "Choose the type of build. Options are: None, Debug, Release, RelWithDebInfo, MinSizeRel." 
      FORCE 
)
```

Reply:
@BSolonenko You can correct it, should run cmake-format with follow argument: --dangle-parens 1 or add in config-file as dangle_parens: True

Reply:
@MaximRybak please make this default